lib/elixir/lib/inspect/algebra.ex

```diff
@@ -565,6 +599,10 @@ defmodule Inspect.Algebra do
     end
   end
 
+  defp ansi(color) do
+    color |> IO.ANSI.format_fragment(true)
```

Comment:
I don't think we should rely on state deep inside Inspect.Algebra. The IO ANSI enabled stateful check should be done at the boundary. Otherwise this would be the only non-pure function in the whole module (which would creep all the way up to rendering too).